zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/job/JobCompleteProcessor.java

```diff
@@ -51,21 +54,31 @@ public final class JobCompleteProcessor implements CommandProcessor<JobRecord> {
           (job key '%d', type '%s', processInstanceKey '%d'). \
           Only the following listener event types support denying: %s.
           """;
-
   private static final String TL_JOB_COMPLETION_WITH_DENY_AND_CORRECTIONS_NOT_SUPPORTED_MESSAGE =
       """
           Expected to complete task listener job with corrections, but the job result is denied \
           (job key '%d', type '%s', processInstanceKey '%d'). \
           The corrections would be reverted by the denial. Either complete the job with corrections \
           without setting denied, or complete the job with a denied result but no corrections.
           """;
-
   private static final String TL_JOB_COMPLETION_WITH_UNKNOWN_CORRECTIONS_NOT_SUPPORTED_MESSAGE =
       """
           Expected to complete task listener job with a corrections result, but property '%s' \
           cannot be corrected (job key '%d', type '%s', processInstanceKey '%d'). \
           Only the following properties can be corrected: %s.
           """;
+  private static final String
+      TL_JOB_COMPLETION_WITH_ASSIGNEE_CORRECTION_ON_CREATING_NOT_SUPPORTED_MESSAGE =
+          """
+          Expected to complete task listener job, but correcting the assignee on 'CREATING' event is \
+          not supported when the user task has an assignee defined in the model \
+          (job key '%d', type '%s', processInstanceKey '%d').
```

Comment:
🔧 Let's expand this a bit more to be helpful.

>Expected to complete task listener job, but correcting the assignee on 'CREATING' event is not supported when the user task has an assignee defined in the model. Use the 'ASSIGNING' event to correct the defined assignee.

💭 We could also clarify both use cases, but it might lead to confusion about the current action to take. What do you think?

>Expected to complete task listener job, but correcting the assignee on 'CREATING' event is not supported when the user task has an assignee defined in the model. Use the 'CREATING' event to set an assignee when it's not defined. Use the 'ASSIGNING' event to correct the defined assignee.